CHANGELOG.md

```diff
@@ -20,7 +20,7 @@
 
```

Comment:
you should add the placeholder for the next [Unreleased]
or is the plan to add it in a follow up PR?

Reply:
Yes, there'll be a follow-up PR called "Post Release"